internal/v1/handler_blueprints.go

```diff
@@ -120,11 +125,31 @@ func (h *Handlers) GetBlueprint(ctx echo.Context, id openapi_types.UUID) error {
 		}
 		return err
 	}
+
 	blueprint, err := BlueprintFromEntry(blueprintEntry)
 	if err != nil {
 		return err
 	}
 
+	if ctx.Request().Header.Get("Accept") == "application/json;export=true" ||
+		ctx.Request().Header.Get("Content-Type") == "application/json;export=true" {
+		if blueprint.Customizations.Subscription != nil {
+			blueprint.Customizations.Subscription.ActivationKey = ""
+		}
+
+		blueprintExportResponse := BlueprintExportResponse{
+			Name:           blueprintEntry.Name,
+			Description:    blueprintEntry.Description,
+			Distribution:   blueprint.Distribution,
+			Customizations: blueprint.Customizations,
+			Metadata: BlueprintMetadata{
+				ExportedAt: time.Now().String(),
```

Comment:
This will work, I would probably suggest to return UTC. This will return a string in server timezone (but including TZ info so it is semantically correct).